core/src/table/table.tsx

```diff
@@ -29,6 +29,8 @@ export interface TableColumn<R> {
 		| ((row: R) => ReactNode); // Render function;
 }
 
+export type TableSize = "small" | "medium" | "large";
```

Comment:
In Moai we don't prefer "string"-based options, but "object"-based, so that Moai's theme can extend them.

You can see how we do it in button here:

https://github.com/moaijs/moai/blob/39ee98c7afb8b0fbc30f0fb003afe018b89414d2/core/src/button/button.tsx#L18-L22

https://github.com/moaijs/moai/blob/39ee98c7afb8b0fbc30f0fb003afe018b89414d2/core/src/button/button.tsx#L172-L184